<launch>

    <node pkg="image_proc" type="image_proc" name="image_proc" ns="/camera" output="screen" />
    
    <node pkg="spiri_optical_flow" type="dense_optflow_node" name="optflow" output="screen">
        <param name="cam_topic" value="/camera/image_raw" />
        <param name="caminfo_topic" value="/camera/camera_info" />
        <param name="display" value="false" />
        <param name="threshold" value="0.0" />
    </node>

</launch>
